test/manual/amp-consent/amp-consent-reprompt.html

```diff
@@ -0,0 +1,218 @@
+<!doctype html>
```

Comment:
Thanks for the manual test! Maybe we could make this test-page include buttons to test all interactions?

Reply:
I think we do have all the actions in this example. We have `accept` `reject` and `dismiss` in the UI. And the two versions of `prompt()` and `prompt(expireCache=true)`.

Any other interactions to include?

Reply:
Oops, guess I didn't see that. Then maybe it's worth renaming the file to `amp-consent-actions.html` or something similar.

Reply:
sg. done